node/device/transport/http/lib/http.js

```diff
@@ -295,7 +295,7 @@ Http.prototype.getReceiver = function getReceiver (done) {
     this._receiver = new HttpReceiver(this._config, this._http);
   }
   
-  done(this._receiver);
+  done(null, this._receiver);
```

Comment:
yes. all our callbacks should follow the nodeback-style (err, result) parameters if we want to be consistent and enable easy promisification